modules/boulder/src/main/ts/ephox/boulder/api/ValueSchema.ts

```diff
@@ -14,85 +15,54 @@ export interface SchemaError<T> {
 
 const _anyValue: Processor = value(SimpleResult.svalue);
 
-const arrOfObj = (objFields: ValueProcessorAdt[]): Processor => {
-  return _arrOfObj(objFields);
-};
+const arrOfObj = (objFields: ValuePresence.ValueProcessorTypes[]): Processor => _arrOfObj(objFields);
 
-const arrOfVal = (): Processor => {
-  return arrOf(_anyValue);
-};
+const arrOfVal = (): Processor => arrOf(_anyValue);
 
 const valueThunkOf = valueThunk;
 
-const valueOf = (validator: (a: any) => Result<any, any>): Processor => {
-  return value((v) =>
-    // Intentionally not exposing "strength" at the API level
-    validator(v).fold<any>(SimpleResult.serror, SimpleResult.svalue)
-  );
-};
+const valueOf = (validator: (a: any) => Result<any, any>): Processor =>
+  value((v) => validator(v).fold<any>(SimpleResult.serror, SimpleResult.svalue));
```

Comment:
Can we keep the comment in here? Or is it no longer valid

Reply:
No longer valid, as @metricjs removed `strength` entirely.

Reply:
🙌